Restaurant Page Project Ready for the 1st review.

thsvr opened the pull request.

thsvr commented:
Hi TSE :wave::skin-tone-2: 

Can you please check my project and provide feedback?

Summary: 
-contents of each ‘tab’ inside of its own module;
- tab-switching logic inside of index.js;
- contact and menu tabs as required.

Thank you in advance :)

Regards,

Thaís Viera

MauricioRobayo commented on src/index.js:
```diff
@@ -0,0 +1,52 @@
+  <div class='d-flex justify-content-end credits'/>Icons made by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></div>
+</footer>`;
+
+  document.getElementById('content').innerHTML += '<div id="tabContent"></div>';
```
Why appending this after, you just used a template string, you can include it there: 

```js
innerHTML = `<nav class="navbar navbar-expand-lg bg-rose">
  ...
</footer><div id="tabContent mayriosd"></div>`;
  Homepage.renderHomepage();
```

thsvr commented:
Hi TSE :wave: 

I did all the requirements requested by Mauricio. Can you please check if my project is able to be approved?

Summary of changes: 
- Refactored index.js, homepage, contact page, menu page and location page using createElement and appendChild; 
- Added better description and author's name to JSON file; 
- Reinstall npm for as a devDep;
- Removed lodash and bootstrap from npm;
- Deleted non necessary script code for Bootstrap;

ps. I will update the live version after the project is approved.

Thank you in advance :)

Regards,

Thaís Viera
